grim.go

```diff
@@ -146,6 +151,11 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	return len(url) >= 12 && url[len(url)-12:] != "000000000000"
```

Comment:
@bhand-mm can you verify this is the best way to determine whether a hook has to do with a deleted branch?